packages/prisma/migrations/20240723105132_add_single_use_links_to_event_types/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
```

Comment:
You can remove this migration and the other one because this adds destroyOnUser column, and the other one just removes the destroyOnUse column. Instead run a clean migration and create only a single migration file.